### PR TITLE
Multi account support

### PR DESCRIPTION
**Pull request by typfel**
Updates the UI for the multi account support added in SE.

### CODE REVIEW

File: Wire-iOS/Sources/AppController.m [hunk 1]
```diff
@@ -435,8 +435,11 @@ - (void)setupUserSession:(ZMUserSession *)userSession
     (void)[Settings sharedSettings];
     
     _zetaUserSession = userSession;
-
-    self.analyticsEventPersistence = [[ShareExtensionAnalyticsPersistence alloc] initWithSharedContainerURL:userSession.sharedContainerURL];
+    
+    NSString *appGroupIdentifier = NSBundle.mainBundle.appGroupIdentifier;
+    NSURL *sharedContainerURL = [NSFileManager sharedContainerDirectoryForAppGroupIdentifier:appGroupIdentifier]; // TODO jacob fallback
```
Comment (daehn) [inline Wire-iOS/Sources/AppController.m]: AFAIK this method already handles the fallback internally.

File: Wire-iOS/Sources/AppController.m [hunk 2]
```diff
@@ -447,10 +450,28 @@ - (void)setupUserSession:(ZMUserSession *)userSession
     // Singletons
     AddressBookHelper.sharedHelper.configuration = AutomationHelper.sharedHelper;
     
-    [MessageDraftStorage setupSharedStorageAtURL:userSession.sharedContainerURL error:nil];
+    [MessageDraftStorage setupSharedStorageAtURL:sharedContainerURL error:nil];
```
Comment (daehn) [inline Wire-iOS/Sources/AppController.m]: This means we will share drafts between accounts, but I personally think that's good (as drafts are excluded from the next release anyway this does not matter at the moment, but should be considered when the time has come).

File: Wire-iOS Share Extension/ShareExtensionViewController.swift [hunk 3]
```diff
@@ -106,12 +106,19 @@ class ShareExtensionViewController: SLComposeServiceViewController {
 
     private func recreateSharingSession() {
         let infoDict = Bundle.main.infoDictionary
-
-        guard let applicationGroupIdentifier = infoDict?["ApplicationGroupIdentifier"] as? String,
-            let hostBundleIdentifier = infoDict?["HostBundleIdentifier"] as? String else { return }
+    
+        guard
```
Comment (daehn) [inline Wire-iOS Share Extension/ShareExtensionViewController.swift]: We have an extension on `Bundle` somewhere directly exposing these infoDict values as optional properties, but maybe not accessible from the share extension at the moment.

File: Wire-iOS/Sources/AppController.m (hunk 1 above)
Comment (typfel) [inline Wire-iOS/Sources/AppController.m]: Ah ok

File: Wire-iOS Share Extension/ShareExtensionViewController.swift (hunk 3 above)
Comment (typfel) [inline Wire-iOS Share Extension/ShareExtensionViewController.swift]: I think it's accessible. I'll update

Comment (typfel) [inline Wire-iOS Share Extension/ShareExtensionViewController.swift]: It's not

**Pull request merged**